napari/layers/shapes/_tests/test_shapes_mouse_bindings.py

```diff
@@ -594,6 +594,75 @@ def test_after_in_add_mode_shape(mode, create_known_shapes_layer):
     assert len(layer.selected_data) == 0
 
 
+@pytest.mark.parametrize(
+    'mode',
+    [
+        'add_polygon',
+        'add_path',
+    ],
+)
+def test_clicking_the_same_point_is_not_crashing(
+    mode, create_known_shapes_layer
+):
+    layer, n_shapes, _ = create_known_shapes_layer
+
+    layer.mode = mode
+    position = tuple(layer.data[0][0])
+
+    for _ in range(2):
+        event = read_only_mouse_event(type='mouse_press', position=position)
+        mouse_press_callbacks(layer, event)
+
+        event = read_only_mouse_event(type='mouse_release', position=position)
+        mouse_release_callbacks(layer, event)
+
+    # If there was no move event between the two clicks, we expect value[1] must be None
+    assert layer.get_value(event.position, world=True)[1] is None
```

Comment:
Is this a sufficient test for the concern you raised earlier @melonora?

Reply:
I suspect it is not, because it passes without any changes to the code, but I don't quite understand what we need to check if not this exact thing. Can you clarify things for me?